docs/reference/indices/templates.asciidoc

```diff
@@ -15,7 +15,7 @@ For example:
 --------------------------------------------------
 PUT _template/template_1
 {
-  "template": "te*",
+  "template": ["te*"],
```

Comment:
I might make this one `["te*", "bar*"]` but then leave the others as strings rather than lists.